### DISCUSSION TITLE
Search hint icon colour

### DISCUSSION THREAD

**Discussion by andokai** [software-mansion/react-native-screens#1861] [Ideas]
Hi, so I'm looking to change colour of the Android search hint icon so that it's not invisible in dark mode. I can't find any mention of it in the docs.

Can anyone help?

<img width="433" alt="CleanShot 2023-08-07 at 07 20 25@2x" src="https://github.com/software-mansion/react-native-screens/assets/3984138/fa657522-0b39-453c-a18f-a84b1f900b74">
<img width="433" alt="CleanShot 2023-08-07 at 07 20 47@2x" src="https://github.com/software-mansion/react-native-screens/assets/3984138/52bca3eb-dc48-4451-874a-09bc8120ad8a">

**Comment by kkafar** [comment]
Out of back of my head: there are `hintTextColor` / `headerIconColor` props on Android, however I do not remember exact interaction (whether they change color of `magnifying glass icon` or not). Have you tried using them? 

If these don't work I believe there is no direct way to achieve this, but maybe someone from community has some workaround.

**Reply by andokai** [reply]
Hi, thanks, I've used both of those, which is why "Search" and the close icon are visible in the dark screenshot.

**Comment by guihendias** [comment]
Still a problem, not being able to change the icon color
